storage/StorageBlock.cpp

```diff
@@ -340,22 +340,16 @@ void StorageBlock::sample(const bool is_block_sample,
 }
 
 void StorageBlock::select(const vector<unique_ptr<const Scalar>> &selection,
-                          const Predicate *predicate,
-                          InsertDestinationInterface *destination) const {
+                          InsertDestinationInterface *destination,
+                          const TupleIdSequence *filter) const {
```

Comment:
Can we move the ``filter`` argument before ``destination``, as per Google style guide?